CONTRIBUTING.md

```diff
@@ -44,9 +44,26 @@ Other notes:
 
 ## Pull Requests
 
-To accommodate review process we suggest that PRs are categorically broken up. 
+To accommodate review process we suggest that PRs are categorically broken up.
 Ideally each PR addresses only a single issue. Additionally, as much as possible
-code refactoring and cleanup should be submitted as a separate PRs from bugfixes/feature-additions. 
+code refactoring and cleanup should be submitted as a separate PRs from bugfixes/feature-additions.
+
+### Process for Reviewing PRs
+
+All PRs require 2 Reviews before merge. When reviewing PRs please use the following annotations:
+
+- `LGTM` without an explicit approval means that the changes look good, but you haven't pulled down the code, run tests locally and thoroughly reviewed it.
+- `Approval` through the GH UI means that you understand the code, documentation/spec is updated in the right places, you have pulled down and tested the code locally.
```

Comment:
K so I guess the only funny issue, is when you perform a "request changes" the only way to override that is via an approve right? If that's the case then maybe you want to "approve" something in the github sense to indicate that you're satisfied with the changes made, however this doesn't mean you've reviewed it deeply.... For that reason I think we may still need `DeepAck`

Reply:
I think the answer here is only `Approve` if you have fully reviewed the changes. If you `Request Changes` that would mean you need to fully review to `Approve` which seems like a good policy (if you know enough to request changes on the PR, then you should also be able to review it fully). Thoughts?

Reply:
Done!

Reply:
you can also dismiss your review after you request changes.